service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -660,17 +690,18 @@ internal class RevurderingServiceImpl(
                         }
                     }
                     is BeregnetRevurdering.Opphørt -> {
-                        utbetalingService.simulerOpphør(
-                            sakId = beregnetRevurdering.sakId,
-                            saksbehandler = saksbehandler,
-                            opphørsdato = beregnetRevurdering.periode.fraOgMed,
-                        ).mapLeft {
-                            KunneIkkeBeregneOgSimulereRevurdering.KunneIkkeSimulere(it)
-                        }.map {
-                            val simulert = beregnetRevurdering.toSimulert(it.simulering)
-                            revurderingRepo.lagre(simulert)
-                            identifiserFeilOgLagResponse(simulert)
-                        }
+                        // TODO er tanken at vi skal oppdatere saksbehandler her? Det kan se ut som vi har tenkt det, men aldri fullført.
```

Comment:
Tenker vi kan ha et eget kort hvor vi avklarer forventet oppførsel og går over alle kallene til revurdering og søknadsbehandling. For klage så overskriver vi alltid saksbehandler.

Reply:
Enig. Naturlig oppførsel i mitt hode er at dette oppdateres hver gang vi gjør endringer.